src/modules/add_remove_task.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { fromEvent, debounceTime } from 'rxjs';
+import Todo from './todo.model.js';
+
+// eslint-disable-next-line import/no-mutable-exports
+let TodoList = [];
```

Comment:
- [ ] Let's try to not disable Linter checks in our codebase as they help us write better, clean, and professional code. Disabling Lintger checks defies its purpose. Kindly fix all linter issues and resubmit your project for approval 👍🏽